### DISCUSSION TITLE
Display music/now playing information on cast device

### DISCUSSION THREAD

**Discussion by TomCasavant** [music-assistant/support#649] [Feature requests and ideas]
Maybe I missed something, but I'd like to be able to see what's currently playing on my google home screen, right now it just says "Streaming from Music Assistant" with no music/podcast/radio information. Also, if possible add some album art onto that screen as well

**Comment by jazzyisj** [comment]
You haven't missed anything, that is the expected behaviour.  I agree, it would be nice but I'm not sure it's possible.

**Comment by marcelveldt** [comment]
I'm not sure if its possible without even writing some cast receiver app that receives the metadata.
Currently we're using the builtin "default media receiver" which works great for sending the audio stream but we can't send the metadata for each track (only for the whole stream) nor does google cast support "icy metadata".

A possibility however that just popped my mind is to send a separate video/display stream with now playing info.
Not sure how that is possible or how that works but pretty sure that there is some way... you can even cast the HA interface to your cast screen
